flake.nix

```diff
@@ -50,8 +60,12 @@
       packages = forAllSystems (
         system: { 
           container = import ./container/mk-container.nix {pkgs = nixpkgsFor.${system}; 
```

Comment:
The `container` attribute can change to `singularity`!  (Similarly in the README.)

Reply:
yes, this will be done once the docker part is ready

Reply:
I mean there's no reason not to change it already but fine

Reply:
fair point, changed!